service/src/test/java/com/codeforcommunity/processor/TeamsProcessorImplTest.java

```diff
@@ -83,38 +81,72 @@ void team1() {
     // successfully create a team
     @Test
     public void testCreateTeam1() {
-        setup();
         mockDb.addEmptyReturn("SELECT");
-        mockDb.addEmptyReturn("UPDATE");
-        mockDb.addEmptyReturn("DROP/CREATE");
         mockDb.addEmptyReturn("INSERT");
 
-        Timestamp timestamp = new Timestamp(System.currentTimeMillis());
-        ArrayList<String> emailList = new ArrayList<String>(
-                Arrays.asList("ex1@example.com", "ex2@example.com", "ex3@example.com"));
-        CreateTeamRequest teamRequest = new CreateTeamRequest("teamName", "teamBio", 2, timestamp, emailList);
+        Timestamp timestamp = Timestamp.valueOf("2020-05-30 02:00:55.939");
+        List<String> emailList = Arrays.asList("ex1@example.com", "ex2@example.com", "ex3@example.com");
+        CreateTeamRequest teamRequest =
+                new CreateTeamRequest("teamName", "teamBio", 2, timestamp, emailList);
 
         jwtData = new JWTData(1, PrivilegeLevel.STANDARD);
         processor.createTeam(jwtData, teamRequest);
         assertEquals(2, mockDb.timesCalled("SELECT"));
-        assertEquals(0, mockDb.timesCalled("UPDATE"));
         assertEquals(2, mockDb.timesCalled("INSERT"));
-        assertEquals(0, mockDb.getSqlBindings().get("UNKNOWN").size());
-        assertEquals(0, mockDb.getSqlBindings().get("DROP/CREATE").size());
 
-        verify(emailer, times(3)).sendInviteEmail(anyString(),
-                anyString(), any(Users.class), any(Team.class));
+        ArgumentCaptor<String> stringArgs = ArgumentCaptor.forClass(String.class);
+        ArgumentCaptor<Users> usersArgs = ArgumentCaptor.forClass(Users.class);
+        ArgumentCaptor<Team> teamArgs = ArgumentCaptor.forClass(Team.class);
+        verify(emailer, times(3)).sendInviteEmail(stringArgs.capture(),
+                stringArgs.capture(),
+                usersArgs.capture(), teamArgs.capture());
+
+        List<String> capturedStrings = stringArgs.getAllValues();
+        assertEquals("ex1@example.com", capturedStrings.get(0));
+        assertEquals("Team Member", capturedStrings.get(1));
+        assertEquals("ex2@example.com", capturedStrings.get(2));
+        assertEquals("Team Member", capturedStrings.get(3));
+        assertEquals("ex3@example.com", capturedStrings.get(4));
+        assertEquals("Team Member", capturedStrings.get(5));
+        List<Users> capturedUsers = usersArgs.getAllValues();
+        // code should be fixed
+        //assertEquals("test", capturedUsers.get(0));
+//        assertEquals("null", capturedUsers.get(1));
+//        assertEquals("null", capturedUsers.get(2));
```

Comment:
Wait, is this test working? What is this here?

Reply:
I made a bug report related to this. It seems like createTeam (function) is not adding the user data properly, so the calls return null for now.

Reply:
Can you add the link to the bug report doc here? Also, I think it might be easier to move the bug list over as github issues, but we'll see about doing that after I look at the doc again.